rotkehlchen/tests/unit/test_evm_transactions.py

```diff
@@ -92,3 +99,70 @@ def test_erc20_transfers_range_not_updated_on_remote_error(database: 'DBHandler'
 
     with database.conn.read_ctx() as cursor:  # ensure the range was not marked as pulled
         assert database.get_used_query_range(cursor=cursor, name=location_string) is None
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('optimism_manager_connect_at_start', [(WeightedNode(node_info=NodeName(name='mainnet-optimism', endpoint='https://mainnet.optimism.io', owned=True, blockchain=SupportedBlockchain.OPTIMISM), active=True, weight=ONE),)])  # noqa: E501
+@pytest.mark.parametrize('optimism_accounts', [['0x706A70067BE19BdadBea3600Db0626859Ff25D74']])
+def test_txlist_etc_falls_back_to_blockscout(
+        database: 'DBHandler',
+        optimism_manager: 'OptimismManager',
+        optimism_accounts: list['ChecksumEvmAddress'],
+) -> None:
+    """Test that the txlist, etc which rely on indexers such as etherscan and blockscount
+    properly fall back to blockscout when etherscan fails. Uses optimism since free tier etherscan
```

Comment:
Do you assert this failure somewhere? As they may revert this at some point and this would affect this test.